AbstractSDK/abstract
# Started accepting raw assets in dex adapter'

This PR aims at providing a simple interface that bypasses ANS for abstract-sdk users. 

This needs to be merged after https://github.com/AbstractSDK/abstract/pull/245


### TODO
- [x] Modify the adapter contract to accept assets without ANS
- [x] Test to verify common use cases work 
- [x] Create a functional and simple API for users who have all the addresses already !
   - Use `.ans_dex(..).swap(...)` to swap with ans assets
   - Use `.dex(..).swap(...)` to swap with cw_assets
- [x] Make sure the API is rational and refactor code for simplicity
- [x] Choose a strategy for astrovault adapterand implement it

## Done
- Removed unique_pool_id from fetch data
- Added pool type fetcher for astrovault
- Added possibility to use `PoolAddress::Contract` for Kujira (to have a common api) 

### Checklist
